IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -3057,16 +3053,17 @@ components:
       properties:
         minimalPrice:
           $ref: '#/components/schemas/CurrencyPriceDef'
-        overallServiceClass:
-          $ref: '#/components/schemas/ServiceClassDef'
+        overallServiceClassId:
```

Comment:
we need more details on the interpretation of "overall". As i understood this is only an indication which can optionally be used for connecting offers from sources.

Reply:
Overall XXX is the result of "adding up" all the values of underlying offerParts. I described how these addition should take place in the documentation